packages/web-ui/src/lab/Link/Link.docs.mdx

```diff
@@ -0,0 +1,128 @@
+import { Meta, Canvas, ArgTypes, Description } from '@storybook/blocks';
+import * as Stories from './Link.stories';
+
+<Meta of={Stories} />
+
+# Link
+
+<Description />
+
+<Canvas of={Stories.SimpleExample} />
+
+## Usage
+
+This `Link` component is part of the lab submodule, a way for us to make new
+components available prior to an official major release. This helps us bed in
+new components before introducing breaking changes, and to avoid blocking the
+availability of these components for designers.
+
+There are 2 options for importing this component.
+
+1. Import directly from the `lab` directory.
+
+```tsx
+import { Box } from '@utilitywarehouse/web-ui';
+import { Link } from '@utilitywarehouse/web-ui/dist/lab';
+
+[...]
+
+<Box>
+  <Link>Link</Link>
+</Box>
+```
+
+2. Import `Lab` directly from Web UI.
+
+```tsx
+import { Box, Lab } from '@utilitywarehouse/web-ui';
+
+[...]
+
+<Box>
+  <Lab.Link>Link</Lab.Link>
+</Box>
+// or
+const { Link } = Lab
+<Box>
+  <Link>Link</Link>
+</Box>
+```
+
+## Size
+
+The size prop controls the size of the `Link`. This is a responsive prop and
+can be used to display different sizes at different breakpoints.
+
+<Canvas of={Stories.ResponsiveSize} />
```

Comment:
Could we also include a story with the two sizes (large and small) with icons? 
When I was trying to play around the storybook playground to test it and see the component configuration I couldn't find an accessible way to test the size and the gap  with icons. 

<img width="1122" alt="Screenshot 2023-11-16 at 15 34 59" src="https://github.com/utilitywarehouse/design-systems/assets/1488514/6394a1b5-dbcf-498b-87fc-47b0d869dd80">